app/views/conferences_primer_theme/pages/about.html.haml

```diff
@@ -13,6 +13,9 @@
     - cache [partable_for(:minutes), content(:minutes)&.structure_items,
              content(:minutes)&.structure_items&.collect_concat(&:structure_parts)&.collect(&:structure_partable)] do
       .mt-4.col-12.col-md-6.float-md-left= render partial: 'minutes', object: content(:minutes)
+    - cache [partable_for(:useful_documents), content(:useful_documents)&.structure_items,
+             content(:useful_documents)&.structure_items&.collect_concat(&:structure_parts)&.collect(&:structure_partable)] do
+      .mt-4.col-12.col-md-6.float-md-left= render partial: 'useful_documents', object: content(:useful_documents)
```

Comment:
Currently the column layout you have will give you three half content-width columns floated left. So you'll have two columns on the first row, and one column on a second row on the left (there are twelve column units in total, and at medium and larger screen sizes six are currently being assigned to each column). It might be better to have narrower columns instead so they all fit, in which case you should change `.col-md-6` to `.col-md-4` in lines 12, 15, and 18.
```suggestion
      .mt-4.col-12.col-md-4.float-md-left= render partial: 'useful_documents', object: content(:useful_documents)
```

Reply:
This was actually intended - three columns on one row seems too crowded, especially since some of these docs could have long names.

Reply:
Ah okay, and you think a full-width column on the second row would be too wide?

Reply:
I thought it would be, although having another look now it does look better - I'll change it to full width.